core/src/components/datetime/datetime.tsx

```diff
@@ -714,6 +717,12 @@ export class Datetime implements ComponentInterface {
           return;
         }
 
+        const { month, year, day } = refMonthFn(this.workingParts);
+
+        if (isMonthSwipeDisabled(month, year, this.workingParts, this.minParts, this.maxParts)) {
```

Comment:
Could this just be `isMonthDisabled`? Not sure I understand the purpose of this function.

Reply:
Let me re-evaluate it based on the recent changes to addressing the Webkit IO issue. 

Originally it existed, because I was using `display: none` for the `calendar-month-disabled` class and we needed to render the working month, even if it was disabled (for instances where your current date value was outside of the min/max bounds). 

That may not exist anymore.

Reply:
Ok cool, it's needed for one case with rendering, so that you cannot free scroll the container when all months are disabled, but a day/week within the working month is enabled. Instead of having a util for that single case, I migrated it to the `isMonthDisabled` util and added the check in the `datetime.tsx` render.